source/server/http/admin.cc

```diff
@@ -314,7 +315,7 @@ void AdminImpl::writeClustersAsJson(Buffer::Instance& response) {
       }
     }
   }
-  response.add(MessageUtil::getJsonStringFromMessage(clusters, true)); // pretty-print
+  response.add(MessageUtil::getJsonStringFromMessage(clusters, true, true)); // pretty-print
```

Comment:
Can you add a quick comment on this behavior change? I think if we want to do this, we should probably change it in the `/config_dump` json output, too.

Reply:
Ok. I think then, I lean to revert this for consistency.